src/main/java/com/github/lordrex34/config/util/TimeUtil.java

```diff
@@ -22,109 +22,231 @@
 package com.github.lordrex34.config.util;
 
 import java.time.Duration;
+import java.time.format.DateTimeParseException;
 import java.time.temporal.ChronoUnit;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
- * @author UnAfraid
+ * @author Nik
  */
 public class TimeUtil
```

Comment:
Since this is an utility class, I would suggest making it `final` and giving it a private scoped constructor with comment "utility class"